pages/instructorIssues/instructorIssues.js

```diff
@@ -1,17 +1,91 @@
-var ERR = require('async-stacktrace');
-var _ = require('lodash');
-var express = require('express');
-var router = express.Router();
+const ERR = require('async-stacktrace');
+const _ = require('lodash');
+const moment = require('moment');
+const express = require('express');
+const router = express.Router();
+const SearchString = require('search-string');
 
-var error = require('../../lib/error');
-var paginate = require('../../lib/paginate');
-var sqldb = require('../../lib/sqldb');
-var sqlLoader = require('../../lib/sql-loader');
+const error = require('../../lib/error');
+const paginate = require('../../lib/paginate');
+const sqldb = require('../../lib/sqldb');
+const sqlLoader = require('../../lib/sql-loader');
 
-var sql = sqlLoader.loadSqlEquiv(__filename);
+const sql = sqlLoader.loadSqlEquiv(__filename);
 
 const pageSize = 100;
 
+const commonQueries = {
+    allOpenQuery: 'is:open',
+    allClosedQuery: 'is:closed',
+    allManuallyReportedQuery: 'is:manually-reported',
+};
```

Comment:
add `allAutomaticallyReportedQuery` as `is:automatically-reported`, or whatever the negation of manually-reported is (to match the open/closed pair)?

Reply:
:+1: